xbmc/network/WakeOnAccess.cpp

```diff
@@ -194,7 +188,11 @@ class ProgressDialogHelper
 
     if (m_dialog)
     {
-      m_dialog->SetLine(1, CVariant{line1});
+      m_dialog->SetLine(0, CVariant{line1});
+
+      int nest_level = NestDetect::Level();
+      if (nest_level > 1)
+        m_dialog->SetLine(1, CVariant{ StringUtils::Format("Nesting:%d", nest_level) });
```

Comment:
It is actually sort of a debug-thing that was present at the time this was made. There where threading-issues where some calls done on gui-thread from this class came back recursively and so led to a circular call-stack and potential crash.
I have never seen it kick in so we could remove the entire detection mechanism or do log instead? Perhaps keeping a log is the best way?